server/src/api/modules/groups/services/EditGroupService.ts

```diff
@@ -235,7 +285,7 @@ async function executeUpdate(params: EditGroupParams, updatedGroupFields: Prisma
         data: [
           ...leftEvents,
           ...joinedEvents.map(a => ({ ...a, role: null })),
-          ...changedRoleEvents.map(p => omit(p, 'previousRole'))
+          ...changedRoleEvents.map(p => omit(p, 'previousRole', 'displayName'))
```

Comment:
these are changes from #2 right? have you included all the changes in there? should we close that issue?